### ISSUE TITLE
ExpansionPanel expand icon lacks a label

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#19909]
We will probably need a localized value to label the icon

**Comment by jonahwilliams** [comment]
Instead of doing things, I'm going to apply a merge semantics to combine it with a header
